packages/interactive_media_ads/android/build.gradle

```diff
@@ -51,10 +51,16 @@ android {
 
     dependencies {
         implementation 'androidx.annotation:annotation:1.5.0'
+        implementation 'com.google.ads.interactivemedia.v3:interactivemedia:3.33.0'
         testImplementation 'junit:junit:4.13.2'
         testImplementation 'org.jetbrains.kotlin:kotlin-test'
+        testImplementation "org.mockito.kotlin:mockito-kotlin:4.1.0"
         testImplementation 'org.mockito:mockito-inline:5.1.0'
         testImplementation 'androidx.test:core:1.3.0'
+
+        // org.jetbrains.kotlin:kotlin-bom artifact purpose is to align kotlin stdlib and related code versions.
+        // See: https://youtrack.jetbrains.com/issue/KT-55297/kotlin-stdlib-should-declare-constraints-on-kotlin-stdlib-jdk8-and-kotlin-stdlib-jdk7
+        implementation(platform("org.jetbrains.kotlin:kotlin-bom:1.8.10"))
```

Comment:
This is the same line added to `camera_android_camerax`:  https://github.com/flutter/packages/blob/main/packages/camera/camera_android_camerax/android/build.gradle#L77